webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const Dotenv = require("dotenv-webpack");
+
+module.exports = {
+  entry: "./src/main.tsx",
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx|ts|tsx|mjs)$/,
+        exclude: /node_modules/,
+        loader: "esbuild-loader",
+        options: {
+          target: "es2020",
+        },
+      },
+      {
+        test: /\.svg$/i,
+        issuer: /\.[jt]sx?$/,
+        use: ["@svgr/webpack"],
+      },
+      {
+        test: /\.css$/i,
+        use: ["style-loader", "css-loader"],
+      },
+    ],
+  },
+  resolve: {
+    extensions: [".js", ".jsx", ".ts", ".tsx", ".mjs"],
+    alias: {
+      "@": path.resolve(__dirname, "src/"),
+      "@components": path.resolve(__dirname, "src/components/"),
+      "styled-system": path.resolve(__dirname, "src/styled-system/"),
+    },
+  },
+  output: {
+    filename: "bundle.js",
```

Comment:
요 번들링 된 파일명도 해시값을 담아서 생성해주면 좋을 것 같습니다!

```suggestion
    filename: "[name]_[chunkhash:8].js",
```